samples/KafkaFlow.Sample.Dashboard/KafkaFlow.Sample.Dashboard.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>    
+    <SpaRoot>AngularFiles\</SpaRoot>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\KafkaFlow.Admin.Dashboard\KafkaFlow.Admin.Dashboard.csproj" />
+    <ProjectReference Include="..\..\src\KafkaFlow.Admin.WebApi\KafkaFlow.Admin.WebApi.csproj" />
+    <ProjectReference Include="..\..\src\KafkaFlow.Admin\KafkaFlow.Admin.csproj" />
+    <ProjectReference Include="..\..\src\KafkaFlow.Microsoft.DependencyInjection\KafkaFlow.Microsoft.DependencyInjection.csproj" />
+    <ProjectReference Include="..\..\src\KafkaFlow\KafkaFlow.csproj" />
+  </ItemGroup>  
+
+  <Target Name="DebugNpmInstall" BeforeTargets="Build">
```

Comment:
We need to find a way to remove this config from client projects and insert it into KafkaFlow.Admin.Dashboard csproj file